### PR TITLE
Feature/cart endpoint

### PR DESCRIPTION
**Pull request by delilahrois**
### What'd ya do to the code?
Added GET and POST for the cart endpoint. Add to cart functionality adds the products to the cart endpoint and adjusts quantity property of each item so there are no duplicates. 

### Select One Thing
- [x] This broke nothing
- [ ] This broke something
- [ ] This broke everything

### Type of change
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up code, removing comments etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

### What's next?

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -65,12 +65,52 @@ app.post('/api/v1/images', (req, res) => {
...
+// GET cart items
+app.get('/api/v1/cart', (req, res) => {
+  const cartItems = app.locals.cart
+  // console.log('RequestPARAMS>>>', req.params)
+  // console.log('Response>>>', res)
+  res.json(cartItems)
+})
+
+// POST item to cart
+app.post('/api/v1/cart', (req, res) => {
```
Comment (reganlosey) [inline server.js:83]: Nice, glad we got this part figured out. Nice to know that we can pull apart `request.body` like this if we want to manipulate its properties and then put it back together

File: server.js [hunk 2]
```diff
@@ -79,38 +119,67 @@ app.locals.images = [
...
     color: ['yellow', 'blue', 'green'],
     artist: 'William Andrews Nesfield',
-    type: 'painting'
+    type: 'painting',
+    quantity: '0',
+    price: '25'
   }
 ]
+
+app.locals.cart = [
```
Comment (reganlosey) [inline server.js:168]: pseudocode is love, pseudocode is life

**Review by reganlosey — APPROVED** [review]
All looks good to me, just gotta merge Eddie's back-end changes and then we're gud to go

**Review by edwardkrupicka — APPROVED** [review]
good job ya'll

**Review by tanya2108 — APPROVED** [review]
Great, Thanks

**Pull request merged**